### PR TITLE
Fix annoying console spam

### PR DESCRIPTION
**Pull request by GanerCodes**
fixes stuff like `import os; os(` causing console spam when it gets completions

### CODE REVIEW

File: xonsh/completers/python.py [hunk 1]
```diff
@@ -273,7 +273,7 @@ def python_signature_complete(prefix, line, end, ctx, filter_func):
         return set()
     try:
         sig = inspect.signature(val)
-    except ValueError:
+    except (ValueError, TypeError):
```
Comment (jnoortheen) [inline xonsh/completers/python.py:276]: I am not sure this is the correct way. For deprecation warning, one should [filter](https://stackoverflow.com/questions/879173/how-to-ignore-deprecation-warnings-in-python) it in their config
(.xonshrc)

Comment (GanerCodes) [reply xonsh/completers/python.py:276]: It's a bug in the autocomplete that will spam the console mid-typing

**Comment by jnoortheen** [comment]
is this the issue you were trying to solve https://github.com/xonsh/xonsh/issues/4721 ?

**Comment by GanerCodes** [comment]
> is this the issue you were trying to solve #4721 ?

it solves one of them yes. The os.fork thing is another thing I should of probs made it's own issue

**Review by jnoortheen — APPROVED** [review]

**Comment by jnoortheen** [comment]
Thanks @GanerCodes !

**Pull request merged**